nikhilyates-two/src/app/landing/LandingPage.tsx

```
@@ -1,34 +1,49 @@
+          </div>
+          
         </div>
         {/* smart navigation divs */}
-        <div className='h-full w-max flex flex-col justify-end'>
-          <div className='h-full flex flex-col justify-end'>
-            <div className='flex max-w-max flex-row justify-end gap-1'>
+        <div className='absolute top-1/2 left-1/2 transform -translate-x-1/2 -translate-y-1/2 w-auto flex flex-col justify-center'>
+          <div className='flex flex-col justify-center'>
+            <div className='flex max-w-max flex-row justify-center gap-1'>
               {contents.map((content) => (
                 <NavCard content={content} />
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)